src/Docfx.Build/Conceptual/ConceptualDocumentProcessor.cs

```diff
@@ -104,9 +104,14 @@ public override SaveResult Save(FileModel model)
             FileLinkSources = model.FileLinkSources,
             UidLinkSources = model.UidLinkSources,
         };
-        if (model.Properties.XrefSpec != null)
+
+        if (((IDictionary<string, object>)model.Properties).TryGetValue("XrefSpec", out var value))
```

Comment:
`TryGetValue` logic is required to suppress **Internal** exception when accessing `model.Properties.XrefSpec` first time.